pkg/dazzle/project_test.go

```diff
@@ -446,3 +454,207 @@ func TestProjectChunk_hash(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadProjectFromRefs(t *testing.T) {
+	imgs := map[string]mockRegistryImage{
```

Comment:
/nit

In my mind (and others e.g. [here](https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing) or [here](https://blog.pragmatists.com/test-doubles-fakes-mocks-and-stubs-1a7491dfa3da) mocks come with verification of behaviour.

I've called them fakes in other tests as they didn't have verification but more strictly they are stubs.
As an aside I've used 'impostor' for more feature complete implementations esp. 3rd party backends where they don't have sandboxes and the protocols were non-standard and complex.

Can we be consistent and call them stubs or fakes?